outline.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Serverless perspectives
+subtitle: An outline 
+...
+
+I.  What is serverless?
+    A.  Elucidating tools, patterns, and practices
+        1. Functions as a Service
+            a. Variable cost application models
+        2. Utility computing
+        3. Frameworks
+            a. Serverless
+            b. Others?
+
+II.  Why should I use serverless?
+    A.  Helping you do what you know you should be doing already
+        1. API-driven development
+            a. Platforms as a first-class citizen
+            b. Meta-data analysis and informed decision making
```

Comment:
Agree - I think this is the big idea that, due to the advent of serverless and enterprise-ready public cloud offerings, starts to become a possibility sooner than most may expect.